packages/engine/lib/simulation-control/src/command/create_remove/migration.rs

```diff
@@ -1484,7 +1513,7 @@ pub(super) mod test {
             &agent_schema,
             MemoryId::new(experiment_id.as_uuid()),
         )?;
-        println!("Thru JSON took: {} us", now.elapsed().as_micros());
+        println!("JSON took: {} us", now.elapsed().as_micros());
```

Comment:
I guess this was supposed to be removed?

Reply:
Sorry, I mean the `println!` in general. I don't think we want `println!` in our code at all, at least it should be `debug!` or similar.